pyrefinebio/sample.py

```diff
@@ -119,61 +119,81 @@ def experiments(self, value):
 
     @classmethod
     def get(cls, accession_code):
-        """Retrieve a sample based on its accession code.
+        """Retrieve a Sample based on its accession code.
 
         returns: Sample
 
         parameters:
 
-            accession_code (str): The accession code for the sample to be retrieved.
+            accession_code (str): The accession code for the Sample to be retrieved.
         """
         result = get_by_endpoint("samples/" + accession_code)
         return cls(**result)
 
     @classmethod
     def search(cls, **kwargs):
-        """Retrieve a list of samples based on various filters
+        """Retrieve a list of Samples based on various filters
 
-        returns: list of samples.
+        returns: list of Sample
 
         Parameters:
-            ordering (str):                  which field to use when ordering the results
-            title (str):
-            organism (str):
-            source_database (str):
-            source_archive_url (str):
-            has_raw (str):
-            platform_name (str):
-            technology (str):
-            manufacturer (str):
-            sex (str):
-            age (number):
-            specimen_part (str):
-            genotype (str):
-            disease (str):
-            disease_stage (str):
+            ordering (str): which field to use when ordering the results
+
+            title (str): filter based on the Sample's title
+
+            organism (str): filter based on the Organism that the Sample was taken from
+
+            source_database (str): filter based on the database that the Sample was taken from
```

Comment:
Assuming that this means GEO, SRA, etc.

```suggestion
            source_database (str): filter based on the publicly available repository that the Sample was taken from
```